### PR TITLE
[ADD] meta files

### PR DESCRIPTION
**Pull request by lmignon**
In preparation for the migration of the first modules we need for v8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[![Build Status](https://travis-ci.org/OCA/bank-statement-reconcile.svg?branch=8.0)](https://travis-ci.org/OCA/bank-statement-reconcile)
+[![Coverage Status](https://coveralls.io/repos/OCA/bank-statement-reconcile/badge.png?branch=8.0)](https://coveralls.io/r/OCA/bank-statement-reconcile?branch=8.0)
+
 Odoo modules for statements tasks
 =================================
```
Comment (pedrobaeza) [inline README.md:6]: Please put a description here:

```
Includes modules for:

* Importing bank statements.
* Complete partner on these statements.
* Provide methods for making automatic reconciliation.
```

Comment (lmignon) [inline README.md:6]: It's a diff ;-) the description is already there

**Comment by bwrsandman** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (pedrobaeza) [inline README.md:6]: Sorry, I didn't see it

**Comment by pedrobaeza** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Travis failed because there's no addons now.

**Pull request merged**